from_cpython/Objects/stringlib/formatter.h

```diff
@@ -1419,8 +1419,10 @@ FORMAT_LONG(PyObject *obj,
             STRINGLIB_CHAR *format_spec,
             Py_ssize_t format_spec_len)
 {
-    return format_int_or_long(obj, format_spec, format_spec_len,
-                              long_format);
+    // py3updating: Temparary solution.
```

Comment:
If these are compile-time issues, you could put something like `RELEASE_ASSERT(0, "unimplemented");` to make it even more obvious.